datasets/afvalwijzer/dataset.json

```
@@ -9,7 +9,7 @@
   "homepage": "https://data.amsterdam.nl",
   "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
   "spatialDescription": "Amsterdam",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Geen wijziging in dit bestand:

```suggestion
  "version": "1.0.0",
```